pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter_test.go

```diff
@@ -52,11 +53,15 @@ var _ = Describe("Pod eviction admitter", func() {
 	const (
 		testNamespace = "test-ns"
 		testNodeName  = "node01"
-		testVMIName   = "my-vmi"
```

Comment:
nit: This const could be removed in the first commit, because `libvmi.New()` sets the VMI's name.

Reply:
Done as part of the most [recent force-push](https://github.com/kubevirt/kubevirt/compare/8487cf940f0f903a72d7754a758dd5a3418cf0ec..9350432aad78ba3dad9af9f0f19ce7af452a5f83).